api/src/main/java/org/pmiops/workbench/interceptors/TracingInterceptor.java

```diff
@@ -68,5 +99,11 @@ public void afterCompletion(
       HttpServletRequest request, HttpServletResponse response, Object handler, Exception ex)
       throws Exception {
     ((Scope) request.getAttribute(TRACE_ATTRIBUTE_KEY)).close();
+    Random rand = new Random();
+    for (int i = 0; i < 100; i++) {
```

Comment:
Why do you need to create a random latency value? If you wanted to do that, you needn't use the interceptor.

Reply:
What you could do is make a test endpoint that accepts options for different distributions on different signals...